style.css

```diff
@@ -1511,3 +1532,148 @@
   width: 295px;
   height: 332px;
 }
+
+.footer {
+  position: absolute;
+  background-color: #6070ff;
+  width: 375px;
+  height: 800px;
+  left: 0;
+  top: 4900px;
+  border-top-left-radius: 100px;
+}
```

Comment:
Great job @Kevo-hacker 🤝

- [ ] As stated in the project requirements, you were required to align elements using flexbox instead of the grid which you actually used. You may please align your contact elements using Flexbox.

Reply:
Yeah, sure let me do that, thank you for letting me know which areas I need to make changes